### ISSUE TITLE
poor performance on TACoS dataset

### ISSUE THREAD

**Issue by Psycho-9** [wjun0830/CGDETR#21]
I reproduced your work locally on Video Moment Retrieval  dataset, I used the charades-sta and tacos datasets provided by your homepage link， the result on charades-sta is normal， but the result on tacos is abnormal ,the loss didn't converge and the R1iou0.5 is almost 0 , is this a problem about TACoS dataset?

**Comment by wjun0830** [comment]
We havent faced such an issue.
Have you set clip len appropriately?

**Comment by Psycho-9** [comment]
How to choose a appropriate clip length according to dataset feature ? Now my cli len is 2, is that appropriate？

**Comment by wjun0830** [comment]
Yes 2 is correct for Tacos. You can compare the feature length and the video length (sec).
Which codebase are you using? QD? CG?

**Comment by Psycho-9** [comment]
Actually I'm using UVCOM which uses QD as basecode, but the author didn't reply to me.

**Comment by wjun0830** [comment]
Hmm.. Im sorry that I havent faced such an issue

**Comment by awkrail** [comment]
@Psycho-9 Hi, if you have any troubles, could you try [our repo](https://github.com/line/lighthouse)? I've reproduced TaCoS experiments w/ CG-DETR.
